app/routes/signup.js

```diff
@@ -0,0 +1,25 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default Route.extend({
+  auth: service(),
+  headData: service(),
+
+  needsAuth: false,
+
+  beforeModel() {
+    if (this.get('auth.signedIn')) {
+      this.transitionTo('index');
+    }
+  },
+
+  activate: function () {
+    this.set('headData.useTailwindBase', true);
+    return this._super(...arguments);
+  },
+
+  deactivate() {
+    this.set('headData.useTailwindBase', false);
+    return this._super(...arguments);
+  },
```

Comment:
It feels like these methods worth a mixin, something like `withTailwind`, what do you think?

Reply:
That sounds good to me! :smile: And actually, I've remembered that there already is a mixin for this, so I've updated to use it